tools/versioner/cmd/root.go

```diff
@@ -196,9 +201,22 @@ func forInplaceUpdate(lmv, stage string, mod modinfo.Provider) (string, error) {
 	if err := os.Rename(temp, dest); err != nil {
 		return "", fmt.Errorf("failed to rename '%s' to '%s': %v", temp, dest, err)
 	}
+	// format code
+	if err = formatCode(mod.DestDir()); err != nil {
+		return "", fmt.Errorf("failed to format code: %v", err)
+	}
 	return tag, nil
 }
 
+func formatCode(path string) error {
```

Comment:
Why is this necessary?

Reply:
Running format in after_script for the whole SDK folder is time consuming, therefore I move the format here, and only run in the modified folder

Reply:
The problem now is that the versioner tool is also responsible for formatting the generated code which IMO is not a good separation of concerns.  Given that this perf problem is not present in swagger-to-sdk-v2, please open a bug to track removing this piece of code once we switch to v2.

Reply:
I removed this from this tool. And I talked with Phoenix (owner of automation bot), he will change  the behaviour in v2 to make the after script only run once for every commit. Previously the after script will run multiple times if there are multiple tags changed.